chore: add city first brand logo to landing page

tristanguckenberger opened the pull request:
The purpose of the PR is to replace the Apollos logo on the landing page with the city first logo. Is the sizing and color in the AFTER screenshot suitable?

BEFORE DARK
![IMG_0032](https://user-images.githubusercontent.com/19194337/162252400-574f011f-7a60-4835-8fad-bd0c1b8d2cad.PNG)

BEFORE LIGHT
![IMG_0033](https://user-images.githubusercontent.com/19194337/162253038-e47fdaa4-016c-4495-b8e5-4476bc35bf44.PNG)

AFTER DARK
![IMG_0031](https://user-images.githubusercontent.com/19194337/162252460-dee59d7a-1927-49c2-bb22-738e227d3ed6.PNG)

AFTER LIGHT
![IMG_0029](https://user-images.githubusercontent.com/19194337/162252859-e376edec-63cc-448c-b2b5-07544b23ea77.PNG)

tristanguckenberger committed:
chore: add city first brand logo to landing page

conrad-vanl commented:
Great! @tristanguckenberger what drives this icon?
![Screen Shot 2022-04-07 at 12 29 42 PM](https://user-images.githubusercontent.com/103927/162262203-8c922043-ee27-4739-8d84-ba347301d3ab.png)

Ideally, this would match the icon of the app, and not be different

tristanguckenberger committed:
chore: update brandicon to match app icon; add brand-icon-text for separate header icon

tristanguckenberger commented:
> Great! @tristanguckenberger what drives this icon? ![Screen Shot 2022-04-07 at 12 29 42 PM](https://user-images.githubusercontent.com/103927/162262203-8c922043-ee27-4739-8d84-ba347301d3ab.png)
> 
> Ideally, this would match the icon of the app, and not be different

@conrad-vanl I believe I initially set this up incorrectly, this icon (brand-icon) was being used for both the header and landing icons. 

In the latest commit, I've added a  separate icon for the header (brand-icon-text) and replaced brand-icon for the onboarding landing with the updated icon SVG.

BrandIcon DARK (onboarding landing page)
![IMG_0034](https://user-images.githubusercontent.com/19194337/162271408-9195333a-89b3-498c-8250-9db03de221f5.PNG)

BrandIcon LIGHT (onboarding landing page)
![IMG_0035](https://user-images.githubusercontent.com/19194337/162271427-7b1e480d-44a4-4df8-8fd5-3a26a70ab13e.PNG)

BrandIconText DARK (App Header)
![IMG_0037](https://user-images.githubusercontent.com/19194337/162271763-7190a4f3-f6bd-4b23-9175-8f8e7a697ba5.PNG)

BrandIconText LIGHT (App Header)
![IMG_0036](https://user-images.githubusercontent.com/19194337/162271805-cb618e91-a71f-4e7e-afd0-e3b6f9db0c76.PNG)
